src/app/blog/iroh-dns/page.mdx

```diff
@@ -110,6 +110,6 @@ The solution would be mDNS. This would allow resolving Node IDs in a local netwo
 
 Now that we can discover nodes, can we take it one step further and use a similar system to discover where you can find a specific document or blob?
 
-We have an [existing prototype]([https://github.com/n0-computer/iroh-examples/blob/main/content-discovery](https://github.com/n0-computer/iroh-examples/blob/main/content-discovery/README.md)) in `iroh-examples` that demonstrates how we might solve content discovery. We can likely leverage the `iroh-dns` infrastructure for resolving content, as well as node addresses.
+We have an [existing prototype](https://github.com/n0-computer/iroh-examples/blob/main/content-discovery/README.md) in `iroh-examples` that demonstrates how we might solve content discovery. We can likely leverage the `iroh-dns` infrastructure for resolving content, as well as node addresses.
```

Comment:
we have moved this since to `iroh-experiments`: https://github.com/n0-computer/iroh-experiments/tree/main/content-discovery#readme

Reply:
Oh well, can you edit the PR now? I'm tired.